cucumber-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/cucumber/helpers/Helpers.java

```diff
@@ -167,7 +167,18 @@ public CharSequence apply(List<Row> rows, Options arg1) throws IOException {
                 tableContent += "</tbody></table>";
                 return tableContent;
             }
-        });
-        return handlebar;
+        };
+    }
+
+    private CharSequence checkState(String arg0, String retValue1, String retValue2, String retValue3) {
+        switch (arg0.toLowerCase()) {
+            case "skipped":
```

Comment:
I know this was already in, but do you think we can have these added to `Constants` and replace other occurrences ?